tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -231,3 +231,14 @@ sudo umount $MOUNT_DIR
 gcsfuse --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/gzip/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME
 sudo umount $MOUNT_DIR
+
+# package local_file
+# Run test with static mounting. (flags: --implicit-dirs=true)
+gcsfuse --implicit-dirs=true --rename-dir-limit=3 $TEST_BUCKET_NAME $MOUNT_DIR
```

Comment:
Sorry why did you create local_file package ? we have the same flag set package [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/rename_dir_limit/rename_dir_limit_test.go#L42) can't we define these test on that package?

Reply:
As discussed offline, since these tests have a logical separation from rename_dir_limit tests, let's keep them separate for now.
I'll create a backlog bug for building GCSFuse only once.